.github/workflows/kernel_tests.yaml

```diff
@@ -0,0 +1,77 @@
+name: Test drivers against a matrix of kernels/distros
+
+on:
+  workflow_dispatch:
+    inputs:
+      libsversion:
+        description: libs version to be tested
+        type: string
+        required: false
+        default: master
+  push:
+    branches:
+      - master
+    
+concurrency:
+  group: kernel-tests-master
+  cancel-in-progress: true
+
+jobs:
+  test-kernels:
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture: [X64, ARM64]
+    runs-on: [ "self-hosted", "linux", "${{matrix.architecture}}" ]    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: alacuku/e2e-falco-tests
```

Comment:
This repo will be soon donated to falcosecurity: https://github.com/falcosecurity/evolution/issues/282.

I will make sure to followup after its donation to update this line!